Tensile/Common.py

```diff
@@ -1138,7 +1137,11 @@ def getArchitectureName(gfxName):
     #   1 = 1 WG per CU (default)
     # TENSILE_STREAMK_FIXED_GRID lets you override the default grid size with a specific number
     #   0 = override disabled (default)
-    "StreamK": [0, 1, 2, 3],
+    "StreamK": [0, 1, 2],
+    # Determines if StreamK kernel uses atomics
+    # 0: uses workspace to store partial tiles, accumulate in deterministic fix-up step
+    # 1: uses atomics to accumulate partial tiles
+    "StreamKAtomic": [0, 1],
```

Comment:
Would you set StreamKAtomic to 0 if StreamK is 0 to avoid generating duplicated kernel for SK=0 + SKA=0 and SK=0 and SKA=0?